src/inverseRobot.js

```diff
@@ -25,6 +25,31 @@
  */
 function inverseRobot(robot) {
   // write code here
+
+  /* Мені здається, що в мене якийсь говнокод, прошу поясніть
+    будь-ласка як це можна переробити через Object.hasOwnProperty,
+     якщо так можна. Бо до мене ніяк не доходить */
+
+  const res = {};
+
+  for (const key in robot) {
+    res[robot[key]] = key;
+  }
+
+  const arr = Object.keys(robot).concat(Object.values(robot));
+  const result = [];
+
+  for (const item of arr) {
+    if (!result.includes(item)) {
+      result.push(item);
+    }
+  }
+
+  if (result.length !== arr.length) {
+    return null;
+  }
```

Comment:
if(!res[robot[key]]) {
  return null;
} else {
  res[robot[key]] = key;
}

Reply:
> if(!res[robot[key]]) {
> return null;
> } else {
> res[robot[key]] = key;
> }


I wrote like this:

const res = {};

  for (const key in robot) {
    if (!res[robot[key]]) {
      return null;
    } else {
      res[robot[key]] = key;
    }
  }

return res;

Not all test pass((